metasv/generate_sv_intervals.py

```diff
@@ -93,22 +147,285 @@ def generate_sc_intervals_callback(result, result_list):
     if result is not None:
         result_list.append(result)
 
+def infer_svtype(aln, isize_mean, isize_sd, num_sd=2):
+    min_isize = isize_mean - num_sd * isize_sd
+    max_isize = isize_mean + num_sd * isize_sd
+    if aln.mate_is_unmapped:
+        return "INS"
+    if aln.tid != aln.rnext:
+        return "CTX,INS"
```

Comment:
I would say lets keep it for now. Later with TX and DUP we should resolve ambiguous softclips in a separate function. I will fix it then.